.gitignore

```diff
@@ -1,5 +1,6 @@
 bin/
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.3.146', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```

Reply:
@TravisEz13 this test seems to come up fairly often but it always resolves itself. Do you know if we have an open issue tracking this?

Reply:
I'm not aware of an open issue.

Reply:
I believe it started when me move to catalina.